src/configuration.ts

```diff
@@ -1,24 +1,42 @@
 import { workspace } from 'vscode';
 import * as path from 'path';
 import { existsSync } from 'fs';
+import semver = require('semver');
```

Comment:
Why not using `import ... from ...` notation?

Reply:
It worked and my IDE didn't complain 🙃. You can maybe replace it with `import { semver } from 'node:semver';` but I havent tested that yet (same with path)

Reply:
I see. Let's use the `import from` variant anyway, similarly to other imports.

Reply:
There are tslint rules for things like this, tslint seems deprecated though so we should migrate to eslint. I'll start putting together another PR after this one.

Reply:
That's orthogonal to the issue at hand. Migration to ESLint should happen (in a separate PR), yet these additions need to be corrected before merging.

Reply:
That's what I was saying. I have this fixed locally but can't push rn as I'm on a plane.